sirepo/template/radia.py

```diff
@@ -553,7 +582,7 @@ def _generate_field_integrals(g_id, f_paths):
                 res[p.name][i_type] = radia_util.field_integral(g_id, i_type, p1, p2)
         return res
     except RuntimeError as e:
-        pkdc('Radia error {}', e.message)
+        pkdlog('Radia error {}', e.message)
```

Comment:
This is a pattern which we may want to encapsulate? Returning `error` should be known by something, e.g. http_reply. I think we may want more context, e.g. which simulation. "Radia" is not necessary in the error message as it it is coming out of template.radia and the line/function will be known

Reply:
575512a

I wanted to distinguish between "sirepo radia" and "oleg Radia" - now using "backend" to clarify that